.github/workflows/build-with-bal-test-native.yml

```diff
@@ -10,13 +10,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Set up GraalVM
         uses: graalvm/setup-graalvm@v1
         with:
           version: 'latest'
-          java-version: '11'
+          java-version: '17.0.7'
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
need to update the graalvm version as proposed in https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/307/files

Reply:
Sure Addressed in [13b5764](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/39/commits/13b5764d1d31fb3c9c6f998b13d7e1332332ad84)